code/game/objects/items/stacks/stack.dm

```diff
@@ -39,9 +39,19 @@
 	. = ..(mapload, material)
 	if (!stacktype)
 		stacktype = type
+	if(!singular_name)
+		singular_name = "sheet"
 	if(!plural_name)
 		plural_name = "[singular_name]s"
 
+/obj/item/stack/dropInto(var/atom/destination)
+	var/oldloc = loc
+	. = ..()
+	if(isturf(loc) && ismob(oldloc))
+		merge_with_stacks_at_loc()
+
+/obj/item/stack/proc/merge_with_stacks_at_loc()
```

Comment:
Why the stub?

Reply:
Oh oops, I started adding that before realizing they already had a proc for it.